sources/packages/backend/apps/db-migrations/src/sql/Reports/Update-data-inventory-report.sql

```diff
@@ -0,0 +1,527 @@
+            parent2.supporting_data ->> ''cppLine30800'' as "Parent 2 Canada pension plan contribution (line 30800)",
+            parent2.supporting_data ->> ''cppLine31000'' as "Parent 2 Canada pension plan contribution (line 31000)",
+            parent2.supporting_data ->> ''eiLine31200'' as "Parent 2 Employment Insurance Premium (Line 31200)",
+            parent2.supporting_data ->> ''totalIncomeTaxLine43500'' as "Parent 2 Income tax paid (Line 43500)",
+            parent2.supporting_data ->> ''parentalContributions'' as "Parent 2 contribution to student",
+            parent2.supporting_data ->> ''parentsOtherDependants'' as "Parent 2 dependants other than applicant",
+            jsonb_array_length(parent1.supporting_data -> ''parentDependentTable'') as "Parent 2 number of other dependants",
+            parent2_dependant_number_post_sec_school.count as "Parent 2 number of dependants on post secondary school",
+            parent2_dependant_number_disability.count as "Parent 2 number of dependants with disability",
+            -- Spouse Info
+            partner_users.first_name as "Spouse''s first name",
+            partner_users.last_name as "Spouse''s last name",
+            partner.sin as "Spouse''s SIN",
+            partner.supporting_data ->> ''totalIncome'' as "Partner income",
+            partner.supporting_data ->> ''partnerEmployed'' as "Partner employment during study period (Yes/No)",
+            partner.supporting_data ->> ''partnerCaringForDependant'' as "Partner caring for dependant (Yes/No )",
+            partner.supporting_data ->> ''partnerLivingWithStudent'' as "Partner living with applicant (Yes/No)",
+            partner.supporting_data ->> ''partnerStudentStudyWeeks'' as "Partner in Post-Secondary number of weeks",
+            partner.supporting_data ->> ''partnerReceiveIncomeAssistance'' as "Partner social assistance amount",
+            partner.supporting_data ->> ''willYourPartnerBeInReceiptOfEmploymentInsuranceBenefitsDuringTheYourStudyPeriodPanel'' as "Partner Employment Insurance amount",
+            partner.supporting_data ->> ''partnerTotalPDBenefits'' as "Partner disability benefits amount",
+            partner.supporting_data ->> ''willYourPartnerBePayingACanadaStudentLoanAndOrAProvincialStudentLoanRegularScheduledPaymentsDuringTheYourStudyPeriodPanel'' as "Partner Canada Student Loan amount",
+            partner.supporting_data ->> ''partnerTotalSupport'' as "Partner child support amount",
+            -- Partner Info Self-Reported by Student
+            applications.data ->> ''estimatedSpouseIncome'' as "Partner estimated income self-reported",
+            -- Family Info
+            current_assessment.workflow_data -> ''calculatedData'' ->> ''familySize'' as "Family size",
+            jsonb_array_length(applications.data -> ''dependants'') as "Number of dependant children in PSE",
+            dependant_11_yo_or_disabled.count as "Number of Dep (0-11 y.o. and disabled 12 y.o. or older)",
```

Comment:
`dependant_11_yo_or_disabled` and `dependant_11_yo_or_disabled` are calculated by Camunda and should be consumed from there. If not saved yet should start to be saved and consumed here.

Reply:
To be clear, I would consider the decision and implementation outside the scope of this PR, IMO.